src/batched/KokkosBatched_InverseLU_Serial_Impl.hpp

```diff
@@ -0,0 +1,195 @@
+      static_assert(vector_type::vector_length == 4 || vector_type::vector_length == 8, 
+                    "AVX, AVX2 and AVX512 is supported");
+      const MKL_COMPACT_PACK format = vector_type::vector_length == 8 ?  MKL_COMPACT_AVX512 : MKL_COMPACT_AVX;
+
+      int r_val = 0;
+      if (A.stride_0() == 1) {
+        //mkl_dgetrfnp_compact(MKL_COL_MAJOR, 
+        //                     m, n, 
+        //                     (double*)A.data(), A.stride_1(), 
+        //                     (MKL_INT*)&r_val, format, (MKL_INT)vector_type::vector_length);
+
+        //void mkl_dgetrfnp_compact (MKL_LAYOUT layout, MKL_INT m, MKL_INT n, double * ap, MKL_INT ldap, 
+        //                           MKL_INT * info, MKL_COMPACT_PACK format, MKL_INT nm);
+
+        //void mkl_dgetrinp_compact (MKL_LAYOUT layout, MKL_INT n, double * ap, MKL_INT ldap, double * work, 
+        //                           MKL_INT lwork, MKL_INT * info, MKL_COMPACT_PACK format, MKL_INT nm);
+        mkl_dgetrinp_compact (MKL_COL_MAJOR, n, 
+                              (double*)A.data(), A.stride_1(), 
+                              (double*)W.data(), n*n, 
+                              (MKL_INT*)&r_val, format, (MKL_INT)vector_type::vector_length);
+
+      } else if (A.stride_1() == 1) {
+        //mkl_dgetrfnp_compact(MKL_ROW_MAJOR, 
+        //                     m, n, 
+        //                     (double*)A.data(), A.stride_0(), 
+        //                     (MKL_INT*)&r_val, format, (MKL_INT)vector_type::vector_length);
+        mkl_dgetrinp_compact (MKL_ROW_MAJOR, n, 
+                              (double*)A.data(), A.stride_0(), 
+                              (double*)W.data(), n*n, 
+                              (MKL_INT*)&r_val, format, (MKL_INT)vector_type::vector_length);
+      } else {
+        r_val = -1;
+      }
+      return r_val;
+    }
+#endif
+
+    template<>
+    template<typename AViewType,
+             typename WViewType>
+    KOKKOS_INLINE_FUNCTION
+    int
+    SerialInverseLU<Algo::InverseLU::Unblocked>::
+    invoke(const AViewType &A, 
+           const WViewType &W,
+           const typename MagnitudeScalarType<typename AViewType::non_const_value_type>::type tiny) {
+        static_assert(AViewType::rank == 2, "A should have two dimensions");
+        static_assert(std::is_same<typename AViewType::memory_space, typename WViewType::memory_space>::value, "A and W should be on the same memory space");
+        assert(A.span()*sizeof(typename AViewType::value_type) <= W.span()*sizeof(typename WViewType::value_type));
+
+        typedef typename AViewType::value_type ScalarType;
+
+        int B_stride_0, B_stride_1;
+        if (WViewType::rank == 1) {
+            if (std::is_same<typename WViewType::array_layout, Kokkos::LayoutRight>::value){
+                B_stride_0 = A.extent(1);
+                B_stride_1 = 1;
+            } 
+            else {
```

Comment:
This case does not make sense to me. If workview is contiguous 1D array, it should always has a unit stride.

Reply:
If the 1D view is non-contiguous, why do you assume that the strides follow the same as A ?

Reply:
@kyungjoo-kim 
Usually ```ww``` should follow ````aa`` 
```auto aa = Kokkos::subview(A, k, Kokkos::ALL(),Kokkos::ALL());
auto ww = Kokkos::subview(W, k, Kokkos::ALL());```

then ```ww``` has the same stride 0 as ````aa```
Correct me if I am wrong?

Reply:
The workview is already strided, which means that the strided location of memory is only accessible. Now you try to override the access pattern from A. I don't think it is a good idea to allow strided view for the 1D case. That should be considered as an error.